hrms/hr/doctype/employee_checkin/employee_checkin.py

```diff
@@ -25,7 +25,13 @@ def validate(self):
 
 	def validate_duplicate_log(self):
 		doc = frappe.db.exists(
-			"Employee Checkin", {"employee": self.employee, "time": self.time, "name": ["!=", self.name]}
+			"Employee Checkin",
+			{
+				"employee": self.employee,
+				"time": self.time,
+				"name": ("!=", self.name),
+				"log_type": ("!=", self.log_type),
```

Comment:
```suggestion
				"log_type": self.log_type,
```

As per your use case, duplicate logs will have the same log type?

Reply:
Oh, you're right! 🙈😅